frontend/packages/signless-transactions/src/hooks/use-create-session.ts

```diff
@@ -100,10 +137,13 @@ function useCreateSession(programId: HexString, metadata: ProgramMetadata | unde
     if (!account) throw new Error('Account not found');
     if (!metadata) throw new Error('Metadata not found');
 
-    const messageExtrinsic = getMessageExtrinsic({ DeleteSessionFromAccount: null });
+    const messageExtrinsic = getMessageExtrinsic({
+      DeleteSessionFromAccount: null,
+    });
+
     const txs = [messageExtrinsic];
 
-    const voucher = await getLatestVoucher(key);
+    const voucher = await getLatestVoucher(key); //if account has voucher
```

Comment:
```suggestion
    const voucher = await getLatestVoucher(key);
```

I think no need it this comment, code pretty much is descriptive enough.